projects/netgrif-components-core/src/lib/navigation/breadcrumbs/abstract-breadcrumbs.component.spec.ts

```
@@ -0,0 +1,106 @@
+        logger
+    ) {
+        super(
+            pathService,
+            caseResourceService,
+            activatedRoute,
+            router,
+            dynamicRoutingService,
+            translateService,
+            logger
+        );
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)